### DISCUSSION TITLE
Problems capturing an entire block with Regex

### DISCUSSION THREAD

**Discussion by andres-gimenez** [typst/typst#3718] [Q&A] [answered]
I want to capture all lines starting with ==? with @show regex as follows

``` 
#show regex("=/?(.+)"): it => {
}
```
As you can see here https://typst.app/project/rc8Mu5JIzP_Q18zo3bRPdu  when I include a line break or a mathematical mode, the regular expression doesn't take up the whole paragraph.
 
Any suggestions?

**Comment by laurmaedje** [comment, marked as answer]
This is a long-standing limitation that will be fixed eventually. See https://github.com/typst/typst/issues/86.
